10up/component-accordion
# Fix/issue 23

### Description of the Change

Adds callback argument to several functions as requested in https://github.com/10up/component-accordion/issues/23

### Alternate Designs



### Benefits



Developer working with the plugin should better be able to target changes to specific elements based on the callback information given in the argument.

### Possible Drawbacks



### Verification Process

1. Open the demo with the new code
2. Verify that existing functionality still exists (open/close, keyboard, etc.)
3. Verify that the browser console logs has data being logged for `onOpen`,` onClose`, and `onToggle` that includes the elements for the item interacted with (link, header, content).

### Checklist:



- [x] My updates pass WCAG 2.x accessibility compliance.
- [x] If I added a new feature, I have also added test cases to ensure accessibility compliance.
- [x] I have appropriately bumped the package version to follow SemVer rules (https://semver.org/)



### Applicable Issues

https://github.com/10up/component-accordion/issues/23
